### PR TITLE
Log action callbacks

### PR DESCRIPTION
**Pull request by paulcsmith**
Closes: #604 

screen shot 2018-09-14 at 2 12 16 pm

### CODE REVIEW

File: src/lucky/action_callbacks.cr [hunk 1]
```diff
@@ -120,11 +123,32 @@ module Lucky::ActionCallbacks
...
+    context.add_debug_message("Stopped at #{callback_method_with_color}")
+  end
+
+  # :nodoc:
+  def self.log_continued_callback(
+    context : HTTP::Server::Context,
+    callback_method_name : String
+  ) : Void
+    callback_method_with_color = callback_method_name.colorize(HTTP::Server::Context::DEBUG_COLOR)
+    context.add_debug_message("Ran #{callback_method_with_color}")
```
Comment (edwardloveall) [inline src/lucky/action_callbacks.cr]: What do you think about `Callback: method_name` here instead? Ran is accurate but maybe not as instructive?

Comment (paulcsmith) [reply src/lucky/action_callbacks.cr]: I had that at first, but it made the log *super* noisy, especially when there were 5 or 6 callbacks. I think it is more clear, but once you know what it means I think the shorter version is nicer.

Can you think of a way to make it clear *and* short?

Comment (edwardloveall) [reply src/lucky/action_callbacks.cr]: I think `Ran` is fine then.

Comment (paulcsmith) [reply src/lucky/action_callbacks.cr]: Ok 🏃‍♂️

**Comment by paulcsmith** [comment]
Thanks for the review!

**Pull request merged**